src/site/content/en/blog/modern-web-games/index.md

```diff
@@ -0,0 +1,113 @@
+
+Here's an example of a web game I built using [Construct 3](http://construct.net) that leverages the URL in a fun and engaging way.
+
+![A level editor interface for a game.](./space-board.png)
+
+[Space Board](https://io-space-board.firebaseapp.com) is a very simple game that can be played on either mobile with touch controls or on desktop with keyboard input. The objective is to navigate a maze of obstacles to reach a goal at the end.
+
+How does Space Board leverage the URL in a unique fashion? By encoding the level structure into the URL itself. All levels are defined as a 10 by 10 grid of objects – e.g. walls, enemy turrets, keys, locked doors etc. The URL then lists all the individual grid positions and their contents. A wall is represented by a `W` character. An empty space is an underscore character.
+
+Here's an example:
+
+`https://io-space-board.firebaseapp.com/?gameWorld=_wwwwwwwwww___ww__eww_k__d___ww___ww___ww_wwwww_www_wwwww_www___ww___ww_s_ww_f_ww___ww___wwwwwwwwwwww`
+
+It's ugly but it does the job.
+
+Upon completing a level in Space Board, the player has the opportunity to design their own level using the simple level editor shown above. By enabling players to design their own levels we are giving them the opportunity for personalisation. When a user feels a connection to a game and a sense of ownership via creation and customisation they are more likely to want to share that ‘thing' with the world.
+
+The desire to share a game is the beginning of the viral loop that we are aiming to achieve with our web games. This game design and sharing mechanism is just one example that's possible but there are many other possibilities – I encourage you to watch my talk for further examples!
+
+## Return on investment
+
+At present, there are ultimately two schools of thought with regards to how a game developer can generate revenue through web games:
+
+- Monetizing the games directly
+- Treating them as an acquisition channel
+
+Treating web games as an acquisition channel means leveraging the web version of your native game as a mechanism to get your players hooked and convincing them to download your larger native binary. You then generate revenue with the native platform's built-in payment and billing backends.
+
+Monetization is usually a mixture of advertising and microtransactions. There is still work to be done for the web to compete with native mobile platforms in game advertising. For example, formats like Rewarded Video Ads have been extremely popular for native mobile games for several years and yet we're only now seeing ad networks deploy these formats on the web.
+
+Nonetheless, there are game developers who continue to be successful on the open web through advertising via traditional banner ads and interstitial video ads. Take a look at [Adsense for Games for more information on these formats](https://support.google.com/adsense/answer/1705831?hl=en).
```

Comment:
> Take a look at \[Adsense for Games for more information on these formats\](https://support.google.com/adsense/answer/1705831?hl=en).

Change to:

> Take a look at \[Adsense for Games\](https://support.google.com/adsense/answer/1705831)  for more information on these formats.

`Adsense for Games` gives sufficient context about what the link goes to and that's mostly just a minor cosmetic change to adhere to style guide. But the important change is removing `?hl=en` from the URL because this will force the reader to see the English version of the post.